modules/gcal/settings.go

```diff
@@ -55,6 +56,7 @@ func NewSettingsFromYAML(name string, ymlConfig *config.Config, globalConfig *co
 
 	settings.colors.day = ymlConfig.UString("colors.day", "forestgreen")
 	settings.colors.description = ymlConfig.UString("colors.description", "white")
+	settings.colors.eventTime = ymlConfig.UString("colors.eventTime", "white")
```

Comment:
In the event that someone is already using `descriptionColor` to change the look of the time, we'll need to support that in a backwards-compatible way. 

You can do that by changing this line to:
```
settings.colors.eventTime = ymlConfig.UString("colors.eventTime",ymlConfig.UString("colors.descriptionColor", "white"))
```

Reply:
I now understood your original comment on the issue. I was wondering if `ymlConfig.UString` occurring twice was a typo in that comment. :sweat_smile: 

This is a good idea. Fixed. :ok_hand: